runtime/dart_isolate.cc

```diff
@@ -110,7 +112,9 @@ DartIsolate::DartIsolate(const Settings& settings,
                          std::string advisory_script_entrypoint,
                          ChildIsolatePreparer child_isolate_preparer,
                          fml::closure isolate_create_callback,
-                         fml::closure isolate_shutdown_callback)
+                         fml::closure isolate_shutdown_callback,
+                         const bool is_root_isolate,
```

Comment:
Nit: Why `const` if this passed by value?

Reply:
Done